### DISCUSSION TITLE
북마크 토글 관련

### DISCUSSION THREAD

**Discussion by enyo9rt** [2022-Harmony/NewsCommunity-fFinal#53] [General]
https://github.com/2022-Harmony/NewsCommunity-fFinal/blob/6f73d42b3690c006a929b0bfc8be439dd590be13/static/js/detail.js#L401-L430

**Comment by choichanhyeok** [comment]
- onclick="toggle_bookmark(1fe4acda-d71e-41fb-91a2-4807c1)"
- onclick=toggle_bookmark("1fe4acda-d71e-41fb-91a2-4807c1")
2022-07-14 해결. 
자바 스크립트의 방만함이 만들어낸 오류,

**Reply by enyo9rt** [reply]
ㅋㅋㅋㅋ
